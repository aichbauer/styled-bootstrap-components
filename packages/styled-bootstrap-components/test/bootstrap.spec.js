import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import {
  Alert,
  Badge,
  A,
  Abbr,
  Address,
  Article,
  Aside,
  B,
  Blockquote,
  BaseButton,
  Caption,
  Code,
  Dd,
  Dfn,
  Dialog,
  Dl,
  Dt,
  Fieldset,
  Figcaption,
  Figure,
  Footer,
  H1,
  H2,
  H3,
  H4,
  H5,
  H6,
  Header,
  Hgroup,
  Hr,
  Img,
  Input,
  Kbd,
  Legend,
  Label,
  Main,
  Navigation,
  Ol,
  Optgroup,
  Output,
  P,
  Pre,
  Progress,
  Samp,
  Section,
  Select,
  Small,
  Strong,
  Sub,
  Summary,
  Sup,
  Svg,
  BaseTable,
  Textarea,
  Th,
  Ul,
  Breadcrumb,
  BreadcrumbItem,
  Button,
  ButtonGroup,
  ButtonToolbar,
  Card,
  CardBody,
  CardFooter,
  CardHeader,
  CardImageHeader,
  CardText,
  CardTitle,
  screenSize,
  Container,
  Dropdown,
  DropdownItem,
  DropdownMenu,
  DropdownDivider,
  FormCheck,
  FormCheckInput,
  FormControl,
  FormControlFile,
  FormControlPlainText,
  FormGroup,
  FormInline,
  FormText,
  InputGroup,
  InputGroupAppend,
  InputGroupPrepend,
  InputGroupText,
  Column,
  Row,
  Jumbotron,
  ListGroup,
  ListGroupItem,
  Modal,
  ModalBody,
  ModalContent,
  ModalDialog,
  ModalFooter,
  ModalHeader,
  ModalTitle,
  Nav,
  NavLink,
  Navbar,
  NavbarLink,
  Popover,
  PopoverArrow,
  PopoverBody,
  PopoverHeader,
  Table,
  Tr,
  Tooltip,
  TooltipArrow,
  TooltipInner,
} from '../src';

test('Simply check if all components are imported and exported correctly', () => {
  const alert = renderer.create(<Alert />).toJSON();
  expect(alert).toMatchSnapshot();

  const a = renderer.create(<A />).toJSON();
  expect(a).toMatchSnapshot();

  const badge = renderer.create(<Badge />).toJSON();
  expect(badge).toMatchSnapshot();

  const abbr = renderer.create(<Abbr />).toJSON();
  expect(abbr).toMatchSnapshot();

  const address = renderer.create(<Address />).toJSON();
  expect(address).toMatchSnapshot();
  const article = renderer.create(<Article />).toJSON();
  expect(article).toMatchSnapshot();

  const aside = renderer.create(<Aside />).toJSON();
  expect(aside).toMatchSnapshot();

  const b = renderer.create(<B />).toJSON();
  expect(b).toMatchSnapshot();

  const blockquote = renderer.create(<Blockquote />).toJSON();
  expect(blockquote).toMatchSnapshot();

  const baseButton = renderer.create(<BaseButton />).toJSON();
  expect(baseButton).toMatchSnapshot();

  const caption = renderer.create(<Caption />).toJSON();
  expect(caption).toMatchSnapshot();

  const code = renderer.create(<Code />).toJSON();
  expect(code).toMatchSnapshot();

  const dd = renderer.create(<Dd />).toJSON();
  expect(dd).toMatchSnapshot();

  const dfn = renderer.create(<Dfn />).toJSON();
  expect(dfn).toMatchSnapshot();

  const dialog = renderer.create(<Dialog />).toJSON();
  expect(dialog).toMatchSnapshot();

  const dl = renderer.create(<Dl />).toJSON();
  expect(dl).toMatchSnapshot();

  const dt = renderer.create(<Dt />).toJSON();
  expect(dt).toMatchSnapshot();

  const fieldset = renderer.create(<Fieldset />).toJSON();
  expect(fieldset).toMatchSnapshot();

  const figcaption = renderer.create(<Figcaption />).toJSON();
  expect(figcaption).toMatchSnapshot();

  const figure = renderer.create(<Figure />).toJSON();
  expect(figure).toMatchSnapshot();

  const footer = renderer.create(<Footer />).toJSON();
  expect(footer).toMatchSnapshot();

  const h1 = renderer.create(<H1 />).toJSON();
  expect(h1).toMatchSnapshot();

  const h2 = renderer.create(<H2 />).toJSON();
  expect(h2).toMatchSnapshot();

  const h3 = renderer.create(<H3 />).toJSON();
  expect(h3).toMatchSnapshot();

  const h4 = renderer.create(<H4 />).toJSON();
  expect(h4).toMatchSnapshot();

  const h5 = renderer.create(<H5 />).toJSON();
  expect(h5).toMatchSnapshot();

  const h6 = renderer.create(<H6 />).toJSON();
  expect(h6).toMatchSnapshot();

  const header = renderer.create(<Header />).toJSON();
  expect(header).toMatchSnapshot();

  const hgroup = renderer.create(<Hgroup />).toJSON();
  expect(hgroup).toMatchSnapshot();

  const hr = renderer.create(<Hr />).toJSON();
  expect(hr).toMatchSnapshot();

  const img = renderer.create(<Img />).toJSON();
  expect(img).toMatchSnapshot();

  const input = renderer.create(<Input />).toJSON();
  expect(input).toMatchSnapshot();

  const kbd = renderer.create(<Kbd />).toJSON();
  expect(kbd).toMatchSnapshot();

  const legend = renderer.create(<Legend />).toJSON();
  expect(legend).toMatchSnapshot();

  const label = renderer.create(<Label />).toJSON();
  expect(label).toMatchSnapshot();

  const main = renderer.create(<Main />).toJSON();
  expect(main).toMatchSnapshot();

  const navigation = renderer.create(<Navigation />).toJSON();
  expect(navigation).toMatchSnapshot();

  const ol = renderer.create(<Ol />).toJSON();
  expect(ol).toMatchSnapshot();

  const optgroup = renderer.create(<Optgroup />).toJSON();
  expect(optgroup).toMatchSnapshot();

  const output = renderer.create(<Output />).toJSON();
  expect(output).toMatchSnapshot();

  const p = renderer.create(<P />).toJSON();
  expect(p).toMatchSnapshot();

  const pre = renderer.create(<Pre />).toJSON();
  expect(pre).toMatchSnapshot();

  const progress = renderer.create(<Progress />).toJSON();
  expect(progress).toMatchSnapshot();

  const samp = renderer.create(<Samp />).toJSON();
  expect(samp).toMatchSnapshot();

  const section = renderer.create(<Section />).toJSON();
  expect(section).toMatchSnapshot();

  const select = renderer.create(<Select />).toJSON();
  expect(select).toMatchSnapshot();

  const small = renderer.create(<Small />).toJSON();
  expect(small).toMatchSnapshot();

  const strong = renderer.create(<Strong />).toJSON();
  expect(strong).toMatchSnapshot();

  const sub = renderer.create(<Sub />).toJSON();
  expect(sub).toMatchSnapshot();

  const summary = renderer.create(<Summary />).toJSON();
  expect(summary).toMatchSnapshot();

  const sup = renderer.create(<Sup />).toJSON();
  expect(sup).toMatchSnapshot();

  const svg = renderer.create(<Svg />).toJSON();
  expect(svg).toMatchSnapshot();

  const baseTable = renderer.create(<BaseTable />).toJSON();
  expect(baseTable).toMatchSnapshot();

  const textarea = renderer.create(<Textarea />).toJSON();
  expect(textarea).toMatchSnapshot();

  const th = renderer.create(<Th />).toJSON();
  expect(th).toMatchSnapshot();

  const ul = renderer.create(<Ul />).toJSON();
  expect(ul).toMatchSnapshot();

  const breadcrumb = renderer.create(<Breadcrumb />).toJSON();
  expect(breadcrumb).toMatchSnapshot();

  const breadcrumbItem = renderer.create(<BreadcrumbItem />).toJSON();
  expect(breadcrumbItem).toMatchSnapshot();

  const button = renderer.create(<Button />).toJSON();
  expect(button).toMatchSnapshot();

  const buttonGroup = renderer.create(<ButtonGroup />).toJSON();
  expect(buttonGroup).toMatchSnapshot();

  const buttonToolbar = renderer.create(<ButtonToolbar />).toJSON();
  expect(buttonToolbar).toMatchSnapshot();

  const card = renderer.create(<Card />).toJSON();
  expect(card).toMatchSnapshot();

  const cardBody = renderer.create(<CardBody />).toJSON();
  expect(cardBody).toMatchSnapshot();

  const cardFooter = renderer.create(<CardFooter />).toJSON();
  expect(cardFooter).toMatchSnapshot();

  const cardHeader = renderer.create(<CardHeader />).toJSON();
  expect(cardHeader).toMatchSnapshot();

  const cardImageHeader = renderer.create(<CardImageHeader />).toJSON();
  expect(cardImageHeader).toMatchSnapshot();

  const cardText = renderer.create(<CardText />).toJSON();
  expect(cardText).toMatchSnapshot();

  const cardTitle = renderer.create(<CardTitle />).toJSON();
  expect(cardTitle).toMatchSnapshot();

  expect(screenSize.sm).toBe('576px');

  const container = renderer.create(<Container />).toJSON();
  expect(container).toMatchSnapshot();

  const dropdown = renderer.create(<Dropdown />).toJSON();
  expect(dropdown).toMatchSnapshot();

  const dropdownItem = renderer.create(<DropdownItem />).toJSON();
  expect(dropdownItem).toMatchSnapshot();

  const dropdownMenu = renderer.create(<DropdownMenu />).toJSON();
  expect(dropdownMenu).toMatchSnapshot();

  const dropdownDivider = renderer.create(<DropdownDivider />).toJSON();
  expect(dropdownDivider).toMatchSnapshot();

  const formCheck = renderer.create(<FormCheck />).toJSON();
  expect(formCheck).toMatchSnapshot();

  const formCheckInput = renderer.create(<FormCheckInput />).toJSON();
  expect(formCheckInput).toMatchSnapshot();

  const formControl = renderer.create(<FormControl />).toJSON();
  expect(formControl).toMatchSnapshot();

  const formControlFile = renderer.create(<FormControlFile />).toJSON();
  expect(formControlFile).toMatchSnapshot();

  const formControlPlainText = renderer.create(<FormControlPlainText />).toJSON();
  expect(formControlPlainText).toMatchSnapshot();

  const formGroup = renderer.create(<FormGroup />).toJSON();
  expect(formGroup).toMatchSnapshot();

  const formInline = renderer.create(<FormInline />).toJSON();
  expect(formInline).toMatchSnapshot();

  const formText = renderer.create(<FormText />).toJSON();
  expect(formText).toMatchSnapshot();

  const inputGroup = renderer.create(<InputGroup />).toJSON();
  expect(inputGroup).toMatchSnapshot();

  const inputGroupAppend = renderer.create(<InputGroupAppend />).toJSON();
  expect(inputGroupAppend).toMatchSnapshot();

  const inputGroupPrepend = renderer.create(<InputGroupPrepend />).toJSON();
  expect(inputGroupPrepend).toMatchSnapshot();

  const inputGroupText = renderer.create(<InputGroupText />).toJSON();
  expect(inputGroupText).toMatchSnapshot();

  const column = renderer.create(<Column />).toJSON();
  expect(column).toMatchSnapshot();

  const row = renderer.create(<Row />).toJSON();
  expect(row).toMatchSnapshot();

  const jumbotron = renderer.create(<Jumbotron />).toJSON();
  expect(jumbotron).toMatchSnapshot();

  const listGroup = renderer.create(<ListGroup />).toJSON();
  expect(listGroup).toMatchSnapshot();

  const listGroupItem = renderer.create(<ListGroupItem />).toJSON();
  expect(listGroupItem).toMatchSnapshot();

  const modal = renderer.create(<Modal />).toJSON();
  expect(modal).toMatchSnapshot();

  const modalBody = renderer.create(<ModalBody />).toJSON();
  expect(modalBody).toMatchSnapshot();

  const modalContent = renderer.create(<ModalContent />).toJSON();
  expect(modalContent).toMatchSnapshot();

  const modalDialog = renderer.create(<ModalDialog />).toJSON();
  expect(modalDialog).toMatchSnapshot();

  const modalFooter = renderer.create(<ModalFooter />).toJSON();
  expect(modalFooter).toMatchSnapshot();

  const modalHeader = renderer.create(<ModalHeader />).toJSON();
  expect(modalHeader).toMatchSnapshot();

  const modalTitle = renderer.create(<ModalTitle />).toJSON();
  expect(modalTitle).toMatchSnapshot();

  const nav = renderer.create(<Nav />).toJSON();
  expect(nav).toMatchSnapshot();

  const navLink = renderer.create(<NavLink />).toJSON();
  expect(navLink).toMatchSnapshot();

  const navbar = renderer.create(<Navbar />).toJSON();
  expect(navbar).toMatchSnapshot();

  const navbarLink = renderer.create(<NavbarLink />).toJSON();
  expect(navbarLink).toMatchSnapshot();

  const popover = renderer.create(<Popover />).toJSON();
  expect(popover).toMatchSnapshot();

  const popoverArrow = renderer.create(<PopoverArrow />).toJSON();
  expect(popoverArrow).toMatchSnapshot();

  const popoverBody = renderer.create(<PopoverBody />).toJSON();
  expect(popoverBody).toMatchSnapshot();

  const popoverHeader = renderer.create(<PopoverHeader />).toJSON();
  expect(popoverHeader).toMatchSnapshot();

  const table = renderer.create(<Table />).toJSON();
  expect(table).toMatchSnapshot();

  const tr = renderer.create(<Tr />).toJSON();
  expect(tr).toMatchSnapshot();

  const tooltip = renderer.create(<Tooltip />).toJSON();
  expect(tooltip).toMatchSnapshot();

  const tooltipArrow = renderer.create(<TooltipArrow />).toJSON();
  expect(tooltipArrow).toMatchSnapshot();

  const tooltipInner = renderer.create(<TooltipInner />).toJSON();
  expect(tooltipInner).toMatchSnapshot();
});
