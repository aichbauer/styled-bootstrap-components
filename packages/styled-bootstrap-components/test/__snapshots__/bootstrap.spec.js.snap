// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simply check if all components are imported and exported correctly 1`] = `
<div
  className="sc-bdVaJa dZWWon"
/>
`;

exports[`Simply check if all components are imported and exported correctly 2`] = `
<a
  className="sc-bdVaJa eRiqeT"
/>
`;

exports[`Simply check if all components are imported and exported correctly 3`] = `
<span
  className="sc-bdVaJa hARFHz"
/>
`;

exports[`Simply check if all components are imported and exported correctly 4`] = `
<abbr
  className="sc-bwzfXH hQIZYv"
/>
`;

exports[`Simply check if all components are imported and exported correctly 5`] = `
<address
  className="sc-htpNat IDuKi"
/>
`;

exports[`Simply check if all components are imported and exported correctly 6`] = `
<article
  className="sc-bxivhb imCjfa"
/>
`;

exports[`Simply check if all components are imported and exported correctly 7`] = `
<aside
  className="sc-ifAKCX kXFzZl"
/>
`;

exports[`Simply check if all components are imported and exported correctly 8`] = `
<aside
  className="sc-EHOje chIqkb"
/>
`;

exports[`Simply check if all components are imported and exported correctly 9`] = `
<blockquote
  className="sc-bZQynM jJNEyA"
/>
`;

exports[`Simply check if all components are imported and exported correctly 10`] = `
<button
  className="sc-gzVnrw dvUyjS"
/>
`;

exports[`Simply check if all components are imported and exported correctly 11`] = `
<caption
  className="sc-htoDjs TqXFR"
/>
`;

exports[`Simply check if all components are imported and exported correctly 12`] = `
<code
  className="sc-dnqmqq iEfGQF"
/>
`;

exports[`Simply check if all components are imported and exported correctly 13`] = `
<dd
  className="sc-iwsKbI cLKkzg"
/>
`;

exports[`Simply check if all components are imported and exported correctly 14`] = `
<dfn
  className="sc-gZMcBi fYCSTw"
/>
`;

exports[`Simply check if all components are imported and exported correctly 15`] = `
<dialog
  className="sc-gqjmRU iMlnVN"
/>
`;

exports[`Simply check if all components are imported and exported correctly 16`] = `
<dl
  className="sc-VigVT lhVxpa"
/>
`;

exports[`Simply check if all components are imported and exported correctly 17`] = `
<dt
  className="sc-jTzLTM eCuhBu"
/>
`;

exports[`Simply check if all components are imported and exported correctly 18`] = `
<fieldset
  className="sc-fjdhpX iOghEC"
/>
`;

exports[`Simply check if all components are imported and exported correctly 19`] = `
<figcaption
  className="sc-jzJRlG ioCwbZ"
/>
`;

exports[`Simply check if all components are imported and exported correctly 20`] = `
<figure
  className="sc-cSHVUG cuVHWp"
/>
`;

exports[`Simply check if all components are imported and exported correctly 21`] = `
<article
  className="sc-kAzzGY busMyf"
/>
`;

exports[`Simply check if all components are imported and exported correctly 22`] = `
<h1
  className="sc-chPdSV gsIoNr"
/>
`;

exports[`Simply check if all components are imported and exported correctly 23`] = `
<h2
  className="sc-kgoBCf heAfjH"
/>
`;

exports[`Simply check if all components are imported and exported correctly 24`] = `
<h3
  className="sc-kGXeez hQezwx"
/>
`;

exports[`Simply check if all components are imported and exported correctly 25`] = `
<h4
  className="sc-kpOJdX hpFqPU"
/>
`;

exports[`Simply check if all components are imported and exported correctly 26`] = `
<h5
  className="sc-dxgOiQ dVBofo"
/>
`;

exports[`Simply check if all components are imported and exported correctly 27`] = `
<h6
  className="sc-ckVGcZ jbogWf"
/>
`;

exports[`Simply check if all components are imported and exported correctly 28`] = `
<header
  className="sc-jKJlTe gpxFaj"
/>
`;

exports[`Simply check if all components are imported and exported correctly 29`] = `
<hgroup
  className="sc-eNQAEJ fbMoJW"
/>
`;

exports[`Simply check if all components are imported and exported correctly 30`] = `
<hr
  className="sc-hMqMXs bwkndM"
/>
`;

exports[`Simply check if all components are imported and exported correctly 31`] = `
<img
  className="sc-kEYyzF fPtozN"
/>
`;

exports[`Simply check if all components are imported and exported correctly 32`] = `
<input
  className="sc-kkGfuU iPhqsl"
/>
`;

exports[`Simply check if all components are imported and exported correctly 33`] = `
<kbd
  className="sc-iAyFgw jornwW"
/>
`;

exports[`Simply check if all components are imported and exported correctly 34`] = `
<legend
  className="sc-hSdWYo inyEYY"
/>
`;

exports[`Simply check if all components are imported and exported correctly 35`] = `
<label
  className="sc-eHgmQL kXCcHL"
/>
`;

exports[`Simply check if all components are imported and exported correctly 36`] = `
<main
  className="sc-cvbbAY bUAGEQ"
/>
`;

exports[`Simply check if all components are imported and exported correctly 37`] = `
<nav
  className="sc-jWBwVP fAZsB"
/>
`;

exports[`Simply check if all components are imported and exported correctly 38`] = `
<ol
  className="sc-brqgnP bkLfOv"
/>
`;

exports[`Simply check if all components are imported and exported correctly 39`] = `
<optgroup
  className="sc-cMljjf kGqvBC"
/>
`;

exports[`Simply check if all components are imported and exported correctly 40`] = `
<output
  className="sc-jAaTju bNkYZj"
/>
`;

exports[`Simply check if all components are imported and exported correctly 41`] = `
<p
  className="sc-jDwBTQ fuUvcK"
/>
`;

exports[`Simply check if all components are imported and exported correctly 42`] = `
<pre
  className="sc-gPEVay dcjgLP"
/>
`;

exports[`Simply check if all components are imported and exported correctly 43`] = `
<progress
  className="sc-iRbamj iKLRDT"
/>
`;

exports[`Simply check if all components are imported and exported correctly 44`] = `
<samp
  className="sc-jlyJG etGLca"
/>
`;

exports[`Simply check if all components are imported and exported correctly 45`] = `
<section
  className="sc-gipzik uKEIp"
/>
`;

exports[`Simply check if all components are imported and exported correctly 46`] = `
<select
  className="sc-csuQGl dtcdPU"
/>
`;

exports[`Simply check if all components are imported and exported correctly 47`] = `
<strong
  className="sc-Rmtcm TdETe"
/>
`;

exports[`Simply check if all components are imported and exported correctly 48`] = `
<strong
  className="sc-bRBYWo kfrlKC"
/>
`;

exports[`Simply check if all components are imported and exported correctly 49`] = `
<sub
  className="sc-hzDkRC gkXgGo"
/>
`;

exports[`Simply check if all components are imported and exported correctly 50`] = `
<summary
  className="sc-jhAzac gaHbMu"
/>
`;

exports[`Simply check if all components are imported and exported correctly 51`] = `
<sup
  className="sc-fBuWsC kLrubk"
/>
`;

exports[`Simply check if all components are imported and exported correctly 52`] = `
<svg
  className="sc-fMiknA iOkVP"
/>
`;

exports[`Simply check if all components are imported and exported correctly 53`] = `
<table
  className="sc-dVhcbM bJkNMW"
/>
`;

exports[`Simply check if all components are imported and exported correctly 54`] = `
<textarea
  className="sc-eqIVtm hTPvyW"
/>
`;

exports[`Simply check if all components are imported and exported correctly 55`] = `
<th
  className="sc-fAjcbJ zEahk"
/>
`;

exports[`Simply check if all components are imported and exported correctly 56`] = `
<ul
  className="sc-caSCKo llhhSE"
/>
`;

exports[`Simply check if all components are imported and exported correctly 57`] = `
<ol
  className="sc-bdVaJa gwKakH sc-brqgnP bkLfOv"
/>
`;

exports[`Simply check if all components are imported and exported correctly 58`] = `
<li
  className="sc-bwzfXH kKLPKO"
/>
`;

exports[`Simply check if all components are imported and exported correctly 59`] = `
<button
  className="sc-bdVaJa ipUclB sc-gzVnrw dvUyjS"
/>
`;

exports[`Simply check if all components are imported and exported correctly 60`] = `
<div
  className="sc-bwzfXH BNlwh"
/>
`;

exports[`Simply check if all components are imported and exported correctly 61`] = `
<div
  className="sc-htpNat ffCmMW"
/>
`;

exports[`Simply check if all components are imported and exported correctly 62`] = `
<div
  className="sc-bdVaJa iWnrTB"
/>
`;

exports[`Simply check if all components are imported and exported correctly 63`] = `
<div
  className="sc-bwzfXH ghFkPJ"
/>
`;

exports[`Simply check if all components are imported and exported correctly 64`] = `
<div
  className="sc-htpNat cpPFok"
/>
`;

exports[`Simply check if all components are imported and exported correctly 65`] = `
<div
  className="sc-bxivhb ikVFYW"
/>
`;

exports[`Simply check if all components are imported and exported correctly 66`] = `
<img
  className="sc-ifAKCX faXbQR sc-kEYyzF fPtozN"
/>
`;

exports[`Simply check if all components are imported and exported correctly 67`] = `
<p
  className="sc-EHOje kDfYFG sc-jDwBTQ fuUvcK"
/>
`;

exports[`Simply check if all components are imported and exported correctly 68`] = `
<h4
  className="sc-dnqmqq biQLAy sc-kpOJdX hpFqPU"
/>
`;

exports[`Simply check if all components are imported and exported correctly 69`] = `
<div
  className="sc-bdVaJa cxSKpF"
/>
`;

exports[`Simply check if all components are imported and exported correctly 70`] = `
<div
  className="sc-bdVaJa hbTNoA"
/>
`;

exports[`Simply check if all components are imported and exported correctly 71`] = `
<div
  className="sc-bwzfXH kyGbme"
/>
`;

exports[`Simply check if all components are imported and exported correctly 72`] = `
<div
  className="sc-htpNat eyWxfm"
/>
`;

exports[`Simply check if all components are imported and exported correctly 73`] = `
<div
  className="sc-bxivhb cxJBQo"
/>
`;

exports[`Simply check if all components are imported and exported correctly 74`] = `
<div
  className="sc-bdVaJa ckmHRx"
/>
`;

exports[`Simply check if all components are imported and exported correctly 75`] = `
<input
  className="sc-bwzfXH kpQPhd sc-kkGfuU iPhqsl"
/>
`;

exports[`Simply check if all components are imported and exported correctly 76`] = `
<input
  className="sc-htpNat bIzgow sc-kkGfuU iPhqsl"
/>
`;

exports[`Simply check if all components are imported and exported correctly 77`] = `
<input
  className="sc-EHOje giJpVN sc-kkGfuU iPhqsl"
/>
`;

exports[`Simply check if all components are imported and exported correctly 78`] = `
<input
  className="sc-bZQynM epSHqO sc-kkGfuU iPhqsl"
/>
`;

exports[`Simply check if all components are imported and exported correctly 79`] = `
<div
  className="sc-gzVnrw dAbIbD"
/>
`;

exports[`Simply check if all components are imported and exported correctly 80`] = `
<form
  className="sc-htoDjs dsAaWw"
/>
`;

exports[`Simply check if all components are imported and exported correctly 81`] = `
<strong
  className="sc-dnqmqq dPMQoH sc-Rmtcm TdETe"
/>
`;

exports[`Simply check if all components are imported and exported correctly 82`] = `
<div
  className="sc-iwsKbI lcucxe"
/>
`;

exports[`Simply check if all components are imported and exported correctly 83`] = `
<div
  className="sc-gZMcBi dJchzE"
/>
`;

exports[`Simply check if all components are imported and exported correctly 84`] = `
<div
  className="sc-gqjmRU ikSodu"
/>
`;

exports[`Simply check if all components are imported and exported correctly 85`] = `
<span
  className="sc-VigVT dDwUkb"
/>
`;

exports[`Simply check if all components are imported and exported correctly 86`] = `
<div
  className="sc-bdVaJa foeCXs"
/>
`;

exports[`Simply check if all components are imported and exported correctly 87`] = `
<div
  className="sc-bwzfXH hggJSN"
/>
`;

exports[`Simply check if all components are imported and exported correctly 88`] = `
<div
  className="sc-bdVaJa dQomal"
/>
`;

exports[`Simply check if all components are imported and exported correctly 89`] = `
<ul
  className="sc-bdVaJa fzpmwg sc-caSCKo llhhSE"
/>
`;

exports[`Simply check if all components are imported and exported correctly 90`] = `
<li
  className="sc-bwzfXH iIfNIg"
/>
`;

exports[`Simply check if all components are imported and exported correctly 91`] = `
<div
  className="sc-bdVaJa ccTDVo"
/>
`;

exports[`Simply check if all components are imported and exported correctly 92`] = `
<div
  className="sc-bwzfXH lddWrE"
/>
`;

exports[`Simply check if all components are imported and exported correctly 93`] = `
<div
  className="sc-htpNat fxEHIu"
/>
`;

exports[`Simply check if all components are imported and exported correctly 94`] = `
<div
  className="sc-bxivhb ctYvZG"
/>
`;

exports[`Simply check if all components are imported and exported correctly 95`] = `
<div
  className="sc-ifAKCX gGRfdJ"
/>
`;

exports[`Simply check if all components are imported and exported correctly 96`] = `
<div
  className="sc-EHOje kmXwEX"
/>
`;

exports[`Simply check if all components are imported and exported correctly 97`] = `
<h4
  className="sc-bZQynM dCHhGA sc-kpOJdX hpFqPU"
/>
`;

exports[`Simply check if all components are imported and exported correctly 98`] = `
<nav
  className="sc-bdVaJa eTBMlR sc-jWBwVP fAZsB"
/>
`;

exports[`Simply check if all components are imported and exported correctly 99`] = `
<a
  className="sc-bwzfXH kwDbDW sc-bdVaJa eRiqeT"
/>
`;

exports[`Simply check if all components are imported and exported correctly 100`] = `
<nav
  className="sc-bdVaJa djQjqz sc-jWBwVP fAZsB"
/>
`;

exports[`Simply check if all components are imported and exported correctly 101`] = `
<a
  className="sc-bwzfXH hEQiCg sc-bwzfXH kwDbDW sc-bdVaJa eRiqeT"
/>
`;

exports[`Simply check if all components are imported and exported correctly 102`] = `
<div
  className="sc-bdVaJa eAZntw"
/>
`;

exports[`Simply check if all components are imported and exported correctly 103`] = `
<div
  className="sc-bwzfXH fygUC"
/>
`;

exports[`Simply check if all components are imported and exported correctly 104`] = `
<div
  className="sc-htpNat djXQbQ"
/>
`;

exports[`Simply check if all components are imported and exported correctly 105`] = `
<h4
  className="sc-bZQynM kvVrYk sc-kpOJdX hpFqPU"
/>
`;

exports[`Simply check if all components are imported and exported correctly 106`] = `
<table
  className="sc-bdVaJa jufgRS sc-dVhcbM bJkNMW"
/>
`;

exports[`Simply check if all components are imported and exported correctly 107`] = `
<tr
  className="sc-bwzfXH jgfQOF"
/>
`;

exports[`Simply check if all components are imported and exported correctly 108`] = `
<div
  className="sc-bdVaJa fElwpp"
/>
`;

exports[`Simply check if all components are imported and exported correctly 109`] = `
<div
  className="sc-bwzfXH jNtWFa"
/>
`;

exports[`Simply check if all components are imported and exported correctly 110`] = `
<div
  className="sc-htpNat gRnePJ"
/>
`;
